src/sys/wait.rs

```diff
@@ -40,14 +40,54 @@ libc_bitflags!(
           target_os = "android"))]
 const WSTOPPED: WaitPidFlag = WUNTRACED;
 
+/// Possible return values from `wait()` or `waitpid()`, indicating a state
```

Comment:
This is too long of a first sentence. Please trim it down and put a empty comment line between it and the more expansive description text. If you run `cargo doc --no-deps --open` you'll see why this is necessary. I believe the same applies to the enum variants below.